SmartDeviceLink/SDLStreamingVideoLifecycleManager.m

```diff
@@ -593,6 +593,12 @@ - (void)sdl_didReceiveRegisterAppInterfaceResponse:(SDLRPCResponseNotification *
     if (resolution != nil) {
         self.videoScaleManager.displayViewportResolution = CGSizeMake(resolution.resolutionWidth.floatValue,
                                  resolution.resolutionHeight.floatValue);
+        // workaround for older high resolution displays that don't support scaling
```

Comment:
```suggestion
        // HAX: Workaround for Ford and Lincoln displays with > 800 resolution width or height. They don't support scaling and if we don't do this workaround, they will not correctly scale the view.
```

Reply:
Hi @joeljfischer, I am not able to modify this PR. So, I created another one #1585 with suggested changes in the comment. Please do let me know if it requires any other update.

Reply:
Thanks for the update Ashwin, I will get back to that PR as soon as I can.